From e36c3f5aa6c81669f922c0055a95068c68805f9f Mon Sep 17 00:00:00 2001
From: Tian Yang <tiany@codeaurora.org>
Date: Mon, 26 Oct 2015 14:20:10 -0500
Subject: [PATCH] skbuff_debug: re-activate object when flushing skbs

Since we can flush the SKBs in the recycler, we should
also re-activate them temporarily when sending the object
back through the SKB free functions so we don't errnously
get errors

Change-Id: I8a507586b5d87d52f352de88266f5a372d816152
Signed-off-by: Matthew McClintock <mmcclint@codeaurora.org>
Signed-off-by: Casey Chen <kexinc@codeaurora.org>
---
 net/core/skbuff_recycle.c | 1 +
 1 file changed, 1 insertion(+)

--- a/net/core/skbuff_recycle.c
+++ b/net/core/skbuff_recycle.c
@@ -177,6 +177,7 @@ static void skb_recycler_free_skb(struct
 
 	while ((skb = skb_dequeue(list)) != NULL) {
 		skb_release_data(skb);
+		skbuff_debugobj_activate(skb);
 		kfree_skbmem(skb);
 	}
 }
